flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/dataset/DataSetUnion.scala

```diff
@@ -36,22 +39,21 @@ import scala.collection.JavaConverters._
 class DataSetUnion(
     cluster: RelOptCluster,
     traitSet: RelTraitSet,
-    leftNode: RelNode,
-    rightNode: RelNode,
-    rowRelDataType: RelDataType)
-  extends BiRel(cluster, traitSet, leftNode, rightNode)
+    inputs: JList[RelNode],
+    rowRelDataType: RelDataType,
+    all: Boolean)
```

Comment:
we don't need the `all` parameter because `DataStreamUnion` only supports `UNION ALL` semantics.